transforms/universal/fdedup/ray/pyproject.toml

```diff
@@ -6,10 +6,11 @@ description = "fdedup Ray Transform"
 license = {text = "Apache-2.0"}
 readme = {file = "README.md", content-type = "text/markdown"}
 authors = [
-    { name = "David Wood", email = "dawood@us.ibm.com" },
-    { name = "Boris Lublinsky", email = "blublinsky@ibm.com" },
+    { name = "Nelson Bore", email = "k.nelsonbore@gmail.com" },
+    { name = "Constantin Adam", email = "cmadam@us.ibm.com" },
 ]
 dependencies = [
+    "dpk_fdedup_transform_python==0.2.2.dev1",
```

Comment:
move to requirements.txt and also use dynamic requirements

Reply:
Fixed in commit https://github.com/IBM/data-prep-kit/pull/699/commits/528457c5cc91dad1439c72258be92e8030f45015